### PR TITLE
fix(ProjectUI):Stale data displayed after using the Group filter in Project Advance Search

### PR DESCRIPTION
**Pull request by afsahsyeda**
Signed-off-by: afsahsyeda <afsah.syeda@siemens-healhtineers.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

Description:
After using the `Group` filter in Project Advance Search, the result displayed will be refreshed on switching tab/reloading of page.

Issue: #1753 

### How To Test?
Please refer #1753

### CODE REVIEW

**Comment by akapti** [comment]
Tested the changes, working fine.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java [hunk 1]
```diff
@@ -1584,15 +1580,16 @@ private Map<String, Set<String>> loadFilterMapFromRequest(PortletRequest request
...
-
-    private void addStickyProjectGroupToFilters(PortletRequest request, User user, Map<String, Set<String>> filterMap){
-        String stickyGroupFilter = ProjectPortletUtils.loadStickyProjectGroup(request, user);
-        if (!isNullOrEmpty(stickyGroupFilter)) {
-            String groupFieldName = Project._Fields.BUSINESS_UNIT.getFieldName();
-            filterMap.put(groupFieldName, Sets.newHashSet(stickyGroupFilter));
-            request.setAttribute(groupFieldName, stickyGroupFilter);
-        }
-    }
+// This function was previously called by loadAndStoreStickyProjectGroup in an if block where groupFilterValue==null.
```
Comment (akapti) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java:1583]: please remove the commented code.

**Review by akapti — CHANGES_REQUESTED** [review]
kindly incorporate the review comments.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java (hunk 1 above)
Comment (afsahsyeda) [reply frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java]: Hi @akapti ,

I have removed it. Please check.

**Review by akapti — APPROVED** [review]
looks good to me.
Approved.

**Pull request merged**